scripts/Install-VSCode.ps1

```diff
@@ -129,7 +131,7 @@ param(
     [switch]$LaunchWhenDone
 )
 
-if (!($IsLinux -or $IsOSX)) {
+if ($env:windir -ne $null) {
```

Comment:
For this kind of test, I usually use `if (($PSVersionTable.PSVersion.Major -le 5) -or $IsWindows) {` i.e. if the platform is <= v5 then it *is* Windows.  If it is v6 or higher then `$IsWindows` will be set appropriately.  But there are several different ways to do this check that is valid across versions & platforms. 

I prefer the above test because it uses PowerShell built-in variables.  Env vars are controlled by the user and there is nothing stopping a Linux/macOS user from defining a `windir` env var.

Reply:
Yeah that seems like the right way to test this

Reply:
Yep makes sense. The bug was opened because of StrictMode and I think what Keith said should play nice with it as well.